server/src/rbx_snapshot.rs

```diff
@@ -348,9 +374,29 @@ fn snapshot_imfs_directory<'source>(
         }
     }
 
+    if let Some(meta_ignore_instances) = meta.ignore_unknown_instances {
+        snapshot.metadata.ignore_unknown_instances = meta_ignore_instances;
+    }
```

Comment:
Can we bump this block to be above where we set children? We're already assigning to the snapshot's metadata and already have a change there.